go/minitwit.go

```diff
@@ -285,7 +289,7 @@ func timeline(w http.ResponseWriter, r *http.Request) {
 	var err error
 	user, err = before_request(r)
 
-	if err != nil {
+	if err != nil || checkNilInterface2(user) {
```

Comment:
Needed when checking on returns from querydb as it returns interface{} nil for the user which is != nil